src/smoothFUBAR/restrictedFUBAR.jl

```diff
@@ -0,0 +1,142 @@
+struct LogPosteriorRFF{T} # Quite sure this is not needed since we already have it in smoothFUBAR.jl
```

Comment:
Drop this, as you say.

Reply:
We now allow the user to choose a prior for psi other than a normal one, so new struct was created to